themes/classic/modules/ps_contactinfo/ps_contactinfo-rich.tpl

```diff
@@ -56,7 +56,7 @@
       <div class="data email">
         {l s='Email us:' d='Shop.Theme.Global'}<br/>
       </div>
-      <a href="mailto:{$contact_infos.email}">{$contact_infos.email}</a>
+      {mailto address={$contact_infos.email} encode="javascript"}
```

Comment:
```suggestion
      {mailto address=$contact_infos.email encode="javascript"}
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17562/commits/d5f5be793c60dd6219fa5977f9595289dfa4063c